plugins/catalog-backend/src/service/DefaultLocationService.test.ts

```diff
@@ -29,7 +29,6 @@ describe('DefaultLocationServiceTest', () => {
     listLocations: jest.fn(),
     getLocation: jest.fn(),
   };
-  const locationService = new DefaultLocationService(store, orchestrator);
```

Comment:
Can just restore this right? Rather than repeating it multiple times?

Reply:
It's done.

Reply:
It's done.